.github/workflows/build.yaml

```diff
@@ -37,8 +37,8 @@ jobs:
           tag=${version}-${base}_edge
           yq e -i ".version = \"$tag\"" rockcraft.yaml
           rockcraft pack --verbose
-      - name: Upload locally built ROCK artifact
-        uses: actions/upload-artifact@v3
-        with:
-          name: charmed-spark
-          path: "charmed-spark_*.rock"
+      # - name: Upload locally built ROCK artifact
+      #   uses: actions/upload-artifact@v3
+      #   with:
+      #     name: charmed-spark
+      #     path: "charmed-spark_*.rock"
```

Comment:
**note** This is a step quite time consuming, but I don't see a reason to have this in, given in the integration tests we just upload things directly to the microk8s ctr

Can I just get rid of this? @taurus-forever

Reply:
You need it on the "release/publish" [stage](https://github.com/canonical/charmed-spark-rock/blob/3.3/edge/.github/workflows/publish.yaml#LL30C1-L33C1).